x-pack/plugins/alerting/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "server/**/*.json",
     "public/**/*",
     "common/*"
-  ],
+, "server/rules_client/lib/.ts"  ],
```

Comment:
nit: formatting for this is showing up kind of funky :)

Reply:
Tbh, I'm not sure how it slipped here. I removed it, there was no intention for me to add it